### ISSUE TITLE
Error: Cannot find module '/home/bart/src/ssg/ssg.config.js'

### ISSUE THREAD

**Issue by louwers** [sw-yx/ssg#11]
Afer installing ssg with `yarn add ssg` I get the following error when trying to run `yarn ssg dev`:

```
ssgConfig file ssg.config.js doesnt exist, continuing as regular sapper app
> Cannot find module '/home/bart/src/ssg/ssg.config.js'
Require stack:
- /home/bart/src/ssg/node_modules/ssg/dist/cli-ssg.js
- /home/bart/src/ssg/node_modules/ssg/dist/cli.js
- /home/bart/src/ssg/node_modules/ssg/ssg
Error: Cannot find module '/home/bart/src/ssg/ssg.config.js'
Require stack:
- /home/bart/src/ssg/node_modules/ssg/dist/cli-ssg.js
- /home/bart/src/ssg/node_modules/ssg/dist/cli.js
- /home/bart/src/ssg/node_modules/ssg/ssg
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:17)
    at Function.Module._load (internal/modules/cjs/loader.js:859:27)
    at Module.require (internal/modules/cjs/loader.js:1028:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at readSSGConfig (/home/bart/src/ssg/node_modules/ssg/dist/cli-ssg.js:78:21)
    at /home/bart/src/ssg/node_modules/ssg/dist/cli.js:92:27
    at Generator.next (<anonymous>)
    at fulfilled (/home/bart/src/ssg/node_modules/ssg/dist/cli.js:5:58)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
It's supposed to be zero-config right?

**Comment by swyxio** [comment]
yes, it is supposed to be zero config. weird, i'll try this out

**Comment by swyxio** [comment]
fixed this with 0.0.58 :) accidentally introduced a regression in my work to introduce a "core data" plugin. 

its zero config, but still needs a svelte file to run anything. put an index.svelte in `src/routes` to get going.

thanks so much for the report! keep em coming. i dont do as much work on this as i should bc i only use it for my own blog haha.

**Comment by swyxio** [comment]
also i need to look into setting up tests..
